pilot/pkg/security/authz/builder/BUILD

```diff
@@ -0,0 +1,17 @@
+go_library(
```

Comment:
can you remove this?

Reply:
google use Bazel to buid go internally?

Reply:
oops, yeah, sorry. not sure why it was generated, I didn't do anything...

@zirain it does, but not for OSS code.

Reply:
good to know that, thanks!